### DISCUSSION TITLE
question about lost dogecoin

### DISCUSSION THREAD

**Discussion by ckoretsky** [dogecoin/dogecoin#3061] [Help needed]
hi, im sorry to bother anyone,  i dont know where else to ask help finding my dogecoin.  I have this from coinbase.  and I was running dogecoin core on both my computers at the time,  silly since i know nothing about how it all works.  thanks if anyone can help

email:  ckoretsky@berkeley.edu
![image](https://user-images.githubusercontent.com/21155303/180288551-6a922528-5641-4371-b238-590231e4dbf7.png)

![Screenshot 2022-07-214111806](https://user-images.githubusercontent.com/21155303/180288917-cc82ab72-9450-47ce-99e3-6f4ae82968dc.png)

**Comment by patricklodder** [comment]
I have some questions to better help you:

1. Did Dogecoin Core give you that address? Which version did you use?
2. Do you still have access to the devices you ran Dogecoin Core on at the time? Did you uninstall / remove anything? Change wallet files? 
3. Which operating system do you use?

**Comment by ckoretsky** [comment]
i use windows 10 or 11  I still have access to the computers, although i've rearranged them a bit.  The version I have now is 14.6.0 but I can't see anything except the debug window.  this is what comes up.  
![image](https://user-images.githubusercontent.com/21155303/180940341-94e7a979-b669-40af-bab5-c466b2a50314.png)

**Reply by SandroMarques46** [reply]
Hello, you have to be more specific. When did you create that coinbase account , where did you get that address, what transfers did you do and from where to where, etc... _**Don't share wallet.dat files or private keys with anyone by the way.**_

**Reply by abosaleh82** [reply]
hello friend,

can you help me to back  my dogecoin or transferred if help you will get 2m dogecoin 
contact me on : 
basameer92@gmail.com

**Reply by SandroMarques46** [reply]
Hello, I replied to you [here](https://github.com/dogecoin/dogecoin/discussions/3176)

**Comment by ckoretsky** [comment]
I don't remember .. all I have is the communications from coinbase which I
can forward to you

**Reply by SandroMarques46** [reply]
If it doesn't include sensitive data then yes so that I can try to identify the operations you did at the time and see if I can help you in some way

**Comment by ckoretsky** [comment]
![Screenshot 2022-11-18 103646](https://user-images.githubusercontent.com/21155303/202779012-c9025d8a-699e-4f1f-86fb-9d850b02e0db.png)
![Screenshot 2022-11-18 103811](https://user-images.githubusercontent.com/21155303/202779026-19341287-1983-4c8e-9b3f-136f43b1d432.png)
![Screenshot 2022-11-18 103856](https://user-images.githubusercontent.com/21155303/202779040-bc03f695-4a3b-4edc-9638-65c17e2387e5.png)
![Screenshot 2022-11-18 103958](https://user-images.githubusercontent.com/21155303/202779054-42eb30ab-7dc6-46ab-9764-9362aeef348c.png)
 i couldnt figure out how to forwqard the emails.  I don't remember much of this , and i cant get back to the doge node I had.  its weird.  thanks for your help

**Reply by SandroMarques46** [reply]
On [this transaction (hash 5789...c2ba5)](https://dogechain.info/tx/5789779e045c3e3139ff5233bbd31bbf47adab0931125f752cfde59b713c2ba5) there was a transfer of 39769.56	to DNAbdSq11fsq8nmwRWcrNihscKuWvBmmdv. This wallet has a [balance of 0 doge](https://dogechain.info/address/DNAbdSq11fsq8nmwRWcrNihscKuWvBmmdv) and was last used on 2022-08-17.

On [this transaction (hash 027ca...82d76)](https://dogechain.info/tx/027ca6d12ad3bbdb08f0df371bfd80435c675692e0fc0b977b46a3bbe9682d76) there was a transfer of 35685.60 to D9JVLPJXcU6xL7A7pWWkvKdDSWtfjgembx. This wallet has a [balance of 9091.897 doge](https://dogechain.info/address/D9JVLPJXcU6xL7A7pWWkvKdDSWtfjgembx) and was last used on 2022-11-09.

Finally there's [this transaction (hash 34680...ded21c)](https://dogechain.info/tx/34680d5e9abb3434b09c55e4ee0dba5dacf51712bb1645a18521be9919ded21c) and there was a transfer of 3043.80 to DGNBsRPsgMn26UgLrxLsfAuFXEyDnZdwn4.  This wallet has a [balance of 3043.80 doge](https://dogechain.info/address/DGNBsRPsgMn26UgLrxLsfAuFXEyDnZdwn4) and was only used for that deposit that I just referenced (2022-05-11).

From my interpretation i think we can discard the first 1 address since it has 0 doge and was recently used (+- 1 week ago) and you said you didn't recently touched anything.

The second wallet didn't have any withdraws , only 9 deposits as you can see [again here](https://dogechain.info/address/D9JVLPJXcU6xL7A7pWWkvKdDSWtfjgembx) . Can you indentify if you used this address before and where? Do you recognize the transactions? Did you ask a friend or something to send you dogecoin? If yes then search your message history to find what address you gave to that person or something like that!

Third wallet mentioned may also be the you are looking for [you can see it again here](https://dogechain.info/address/DGNBsRPsgMn26UgLrxLsfAuFXEyDnZdwn4) since it has a 3k doge balance and only 1 deposit.

From what it seems (altought very unclear) it looks you've transfered the doge to coinbase pro instead of your wallet (or maybe a mixture of these 2 operations...). Can you log in to coinbase pro and check the balances there? (**Because Coinbase balances are different than Coinbase Pro balances! It's like a different platform**)

About dogecoin core, is that the main window the program shows after you launch it? If it is then that's really weird... Also you **do not need to open any ports to use dogecoin core wallet functionalities**. Just install it and use it and regurlary save backups of _wallet.dat_ file

**Reply by ckoretsky** [reply]
[Coinbase-5fcfb90ac7d68a152d33fb0f-TransactionsHistoryReport-2022-11-21-20-02-20.pdf](https://github.com/dogecoin/dogecoin/files/10060204/Coinbase-5fcfb90ac7d68a152d33fb0f-TransactionsHistoryReport-2022-11-21-20-02-20.pdf)

**Reply by ckoretsky** [reply]
![image](https://user-images.githubusercontent.com/21155303/203148801-0de9c2e2-b422-43df-bcc3-5b452fff52a3.png)

**Reply by ckoretsky** [reply]
i dont know if this helps

**Reply by ckoretsky** [reply]
![image](https://user-images.githubusercontent.com/21155303/203150361-a7f02b02-5216-4ada-a1df-2344fb249fc8.png)
![image](https://user-images.githubusercontent.com/21155303/203150577-e414211a-4900-49f8-907d-3fbb33712ff9.png)
[account.pdf](https://github.com/dogecoin/dogecoin/files/10060275/account.pdf)

**Reply by ckoretsky** [reply]
i think its the one where i sent 10,000 doge.  so how would i find that wallet?

**Reply by SandroMarques46** [reply]
My conclusions from what you've sent is the following (analyzing _account.pdf_):

`Withdrawal 2022-05-11 17:48:06 UTC -3045.2400000 DOGE` - went to DGNBsRPsgMn26UgLrxLsfAuFXEyDnZdwn4 :
![image](https://user-images.githubusercontent.com/43933237/203160032-212b0461-2be0-4902-8a7f-1cf6a47f6432.png)

`Withdrawal 2021-12-16 19:19:02 UTC -5687.0400000 DOGE` - went to DNAbdSq11fsq8nmwRWcrNihscKuWvBmmdv :
![image](https://user-images.githubusercontent.com/43933237/203160319-2ec392f0-8e56-413a-8153-bec4436b068a.png)

And then there are multiple other withdrawals that i couldn't make a connection with any of the addresses that you mentioned until now.

So for now we need to find where you got **DGNBsRPsgMn26UgLrxLsfAuFXEyDnZdwn4** and **DNAbdSq11fsq8nmwRWcrNihscKuWvBmmdv** from. The second address should probably be easier since as you can see above the most recent transaction (a deposit) was made on 2022-11-09. Did you use dogecoin core on that day? Is it another wallet platform you use? Is it a **Coinbase Pro deposit address** ? ...


The other pdf file (_Coinbase-5fcfb90ac7d68a152d33fb0f-TransactionsHistoryReport-2022-11-21-20-02-20.pdf_) has some addresses and **every single one of the has "pairs" of withdrawals and deposits** , meaning, everytime you receive an amount, you withdraw it to somewhere else, example here :

![image](https://user-images.githubusercontent.com/43933237/203162698-b9686d67-8646-4e20-b0e4-c6929c193432.png)

I think that's a big indication that you're transfering from Coinbase to another exchange/ wallet service and not a personal wallet like dogecoin core

**Reply by ckoretsky** [reply]
i used to trade crypto and i would always move  my tokens across exchanges by converting to doge, because i know we are supposed to use doge as much as possible.  but i havent traded in almost a year i think.  and I only opened up the dogecoin core this month, 2 times and both times i could only get the debug window,  even after uninstalling it.  i never deposited or withdrew any doge in the past 3 months for sure.  from anywhere.  i can't even  get onto any of the exchanges anymore, i finally just got verified to coinbase today. but i know i had 2,000.00 of doge ( when it was 30 cents) in my doge wallet, maybe more.  and i havent seen it at all this year.  can't find my doge wallet, and also, i never had  12 words  because it never told me them.

**Reply by ckoretsky** [reply]
when I right click on the doge icon it won't do show//hide or much receive or much send,  only the four below them
![image](https://user-images.githubusercontent.com/21155303/203206523-977b2d72-1a80-464c-be4a-37746f908cab.png)
, sign message, verify message , options and debug windowoptions,

**Reply by SandroMarques46** [reply]
Yeah it should be something like this 
![image](https://user-images.githubusercontent.com/43933237/203407461-2035a597-f26b-4aa5-bb75-ca82bd7d3662.png)

Very strange that you installed it from scratch and the problem still persists.

My suggestion is grabbing your _wallet.dat_ file and try to install dogecoin core on a different computer and import the wallet (if you want this approach i'll send you a post on reddit that explains how to do it. If you don't know where the folder is, try this path :

C:\Users\<YOUR_WINDOWS_USER_NAME>\AppData\Roaming\ -> Dogecoin/Dogecoin Core
or
Search by "wallet.dat" everywhere

### **Don't show the wallet.dat to anybody non trustworthy! wallet.dat is a file that contains the private/publics keys and everything wallet related. If somebody has access to it they can steal your coins.**

**Reply by SandroMarques46** [reply]
@ckoretsky hello , have you tried it ?

**Comment by ckoretsky** [comment]
![image](https://user-images.githubusercontent.com/21155303/202781755-881c4bbe-3975-43f6-93b5-7d0d3bde2e0c.png)
also i uninstalled and reinstalled the dogecoin core and just now opened it, still i have nothing but the debug window, no send or receive,  i havent had it open for awhile so i don't even remember how to open the ports and all that business

**Reply by ckoretsky** [reply]
![image](https://user-images.githubusercontent.com/21155303/202782207-3b0ae578-4459-4f47-b656-c5e5948cdd12.png)
i guess the port is open

**Comment by abosaleh82** [comment]
Thanks for your reply and helping me i have the the file but i dont now how
to be using it cause need special programer,, can you do me favor to learn
how to resrore the coin and i promes to gives you 20%

**Reply by SandroMarques46** [reply]
I wasn't replying to you , i was replying to ckoretsky ... I'm afraid with your limited english I can't help as much. Maybe try to find somebody that speaks your native language. Either way , if you have the _wallet.dat_ and you want to import it into dogecoin core you can follow this [reddit guide](https://www.reddit.com/r/dogecoin/comments/ld73fe/guide_recover_doges_from_a_walletdat_backup_made/) . Make sure you follow the steps correctly and before doing them , back up some copies of any _wallet.dat_ or sensitive files you might edit ! If it's too hard then you should find someone **very trustworthy** to do it for you. Welcome to crypto.

**Comment by ckoretsky** [comment]
hello and thank you so much for helping me. I finally got the dogecoin core
to load properly so i can see my transactions and it shows that i have
3,000 doge.  I was so relieved that i almost forgot that this is where i
was when i started the search,  i knew i had alot more doge because it was
when i had started emailing coinbase in may of this year that i deposited
that 3,000 doge.  I knew i had another 10,000 somewhere.  So now looking at
your emails i see the wallet that has the 9 deposits you pointed out and
that dogecoin is still there, right?  and I know it's mine so it can't go
anywhere,  i just have to figure out how to get it, right?  So i have this
wallet now,  the one i deposited 3,000 in may.  I have no keys to it and i
don't know how to get the keys,  i mean the 12 words,  if you could help me
there.  Also I have two wallet.dat files i think, one on C drive and one on
D drive.  I remember I had the node running on the
D drive of my computer,  at least i thought that.  and when i looked i had
copies of the program file on both drives.  So I uninstalled them both and
reinstalled them both but now whether i run it started from c or d,  it
always shows just the one deposit of 3,000 doge.


so the 10,000 doge i deposited in december of 21 when i still had money and
was just trading like an idiot,  i added a little more in early this year
but i soon realized what an idiot i was and stopped trading or even looking
at crypto for many months,  so i don't understand these very small deposits
this year,  like the last 6,  i never made those,  i never even saw this
wallet really,  i was just running a node on each of my computers and
transfering doge using the much receive address.  if not for you i would
have just given up and forgotten about that money,  because i've lost every
bit of money i had been trading with, the doge is all i have left.  how
crazy would it be if i actually had a thousand dollars after that whole
trading disaster.  Because coinbase never showed me that wallet like you
did,  i started to think i had imagined the whole thing.  but now i see the
wallet, i know it's mine,  but i still have no idea how to get it.  it's
probably better this way.  i really don't want accesss to it because i know
i will take it out and i will probably regret that alot.  so maybe you
shouldn't help me access it.  but just help me know that if i wait to
figure out how to get it out,  it won't disappear in the meantime.  lol
thanks so much i really really appreciate you.  DOGE TO THE MOON!!!!
